src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -179,6 +179,35 @@ public void Do()
         }
 
         [Fact]
+        public void TestRunEmbeddedStatementNotFollowedBySemicolon()
```

Comment:
Here is it #6676. Please let me know if you mean to add the reference as a comment to the test code.